extensions/amp-auto-ads/0.1/anchor-ad-strategy.js

```diff
@@ -55,22 +44,34 @@ export class AnchorAdStrategy {
    * @return {!Promise<boolean>} Resolves when the strategy is complete.
    */
   run() {
-    if (!this.isAnchorAdEnabled_()) {
+    const filledAnchorEnabled = this.isFilledAnchorAdEnabled_();
+    const noFillAnchorEnabled = this.isNoFillAnchorAdEnabled_();
+
+    if (this.hasExistingStickyAd_()) {
+      if (noFillAnchorEnabled) {
+        user().warn(
+          TAG,
+          'Auto Ads is unable to crawl because of already existing <amp-sticky-ad>.'
```

Comment:
This is not necessarily correct, because the existing sticky ad might already be for the ad network in question, in which case the crawling would still work. Maybe something like 'amp-auto-ads may not work because of already existing &lt;amp-sticky-ad&gt;.'

Reply:
Done. Thanks!!